pygmt/datasets/earth_relief.py

```diff
@@ -120,11 +140,24 @@ def load_earth_relief(resolution="01d", region=None, registration=None, use_srtm
             f"{registration}-registered Earth relief data for "
             f"resolution '{resolution}' is not supported."
         )
-
+    earth_relief_sources = {"igpp": "earth_relief_", "gebco": "earth_gebco_"}
+    if data_source in earth_relief_sources and data_source != "igpp":
```

Comment:
Might need an `else` statement here to catch cases when users didn't use 'igpp' or 'gebco'? Perhaps move the `raise GMTInvalidInput(f"Invalid earth relief 'data_source' ...` line at the bottom here, so that the error is raised earlier.

Reply:
I moved it to its own `if` statement.